reel-picker/src/main/java/com/innowisegroup/reelpicker/picker/ReelPicker.kt

```diff
@@ -87,54 +91,52 @@ class ReelPicker : DialogFragment() {
         )
     }
 
-    private fun setFragmentResultListeners() {
+    private fun setFragmentResultListeners() =
         requireActivity().supportFragmentManager.setFragmentResultListener(
-            UPDATE_TIME_TAB_TITLE_REQUEST_KEY,
+            TAB_TITLE_REQUEST_KEY,
             viewLifecycleOwner
         ) { _, bundle ->
-            val selectedTime =
-                bundle.getSerializable(UPDATE_TIME_TAB_TITLE_KEY) as LocalTime
-            refresh(selectedTime.formatTime())
+            bundle.getSerializable(TAB_VALUE_KEY)?.let { newValue ->
+                when (bundle.getSerializable(TAB_TYPE_KEY) as TabType) {
+                    TIME -> (newValue as LocalTime).also { selectedTime = it }.formatTime()
+                    DATE -> (newValue as LocalDate).also { selectedDate = it }.formatDate()
+                }.also {
+                    tabLayout.getTabAt(viewPager.currentItem)?.text = it
```

Comment:
Don't use viewPager.currentItem. It will cause problems with displaying information in the tab when you use two tabs